src/libtools/linalg.cc

```diff
@@ -19,223 +19,142 @@
 #include <iostream>
 #include <sstream>
 namespace votca { namespace tools {
-
-using namespace std;
-namespace ub = boost::numeric::ublas;
```

Comment:
This is a .cc source file why did you remove using namespace std?

Reply:
do not need it anymore